test/pylib/scylla_cluster.py

```diff
@@ -874,8 +874,12 @@ async def get_cluster_manager(test_name: str, clusters: Pool[ScyllaCluster], tes
         -> AsyncIterator[ScyllaClusterManager]:
     """Create a temporary manager for the active cluster used in a test
        and provide the cluster to the caller."""
-    manager = ScyllaClusterManager(test_name, clusters, test_path)
+    manager: Optional[ScyllaClusterManager] = None
     try:
+        manager = ScyllaClusterManager(test_name, clusters, test_path)
         yield manager
+    except Exception as exc:
+        raise exc
```

Comment:
I didn't understand what's the point of having an "except" clause which catches an exception and throws the same one again. What happens differently if you just drop this "except" line?

Reply:
I read the exception would be swallowed by `asynccontextmanager`, but not I tried a small repro and it's not the case. I'll update.